care/emr/management/commands/test_emr.py

```
@@ -1,6 +1,10 @@
-# ruff : noqa : T201 F841
+# ruff: noqa: T201
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `T201`)

Remove unused `noqa` directive

(RUF100)